From a10573226a21c1ff3be24efdf7f57be3a3108c76 Mon Sep 17 00:00:00 2001
From: Andriy Grytsenko <andrej@rep.kiev.ua>
Date: Mon, 19 Nov 2012 23:38:34 +0200
Subject: [PATCH 14/22] Disabling input for file name in properties dialog
 window.

Changing file name in file properties dialog isn't implemented yet even
for single file therefore field for changing should be incensitive.
---
 src/gtk/fm-file-properties.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/gtk/fm-file-properties.c b/src/gtk/fm-file-properties.c
index ec486f7..ec672b7 100644
--- a/src/gtk/fm-file-properties.c
+++ b/src/gtk/fm-file-properties.c
@@ -786,6 +786,8 @@ static void update_ui(FmFilePropData* data)
         localtime_r(&atime, &tm);
         strftime(buf, sizeof(buf), "%x %R", &tm);
         gtk_label_set_text(data->atime, buf);
+        /* FIXME: changing file name isn't implemented yet, disable entry */
+        gtk_widget_set_sensitive(GTK_WIDGET(data->name), FALSE);
     }
     else
     {
-- 
1.8.0.1

